content/community/conferences.md

```diff
@@ -36,6 +36,11 @@ June 2-4, 2022. Atlanta, GA, USA
 
 [Website](https://renderatl.com) - [Twitter](https://twitter.com/renderATL) - [Instagram](https://www.instagram.com/renderatl/) - [Facebook](https://www.facebook.com/renderatl/) - [LinkedIn](https://www.linkedin.com/company/renderatl)
 
+### App.js Conf 2022 {#appjs-conf-2022}
+June 08-10, 2022. Kraków, PL
```

Comment:
```suggestion
June 08-10, 2022. Kraków, Poland
```

Reply:
https://reactjs.org/community/conferences.html#appjs-conf-2019

We shall have it consistent lets use `Poland` instead of `PL`?

Reply:
Yes, Poland will be the most correct form.